packages/cardpay-sdk/contracts/addresses.ts

```diff
@@ -5,18 +5,18 @@ import { networkName } from '../sdk/utils/general-utils';
 // this silly consumption of the ABI's. The ABI's are indeed consumed, however,
 // we consume them outside of ts due to the way the codegen operates for the
 // subgraph assembly script.
-import PayMerchantHandlerABI from './abi/v0.9.0/pay-merchant-handler';
-import RegisterMerchantHandlerABI from './abi/v0.9.0/register-merchant-handler';
-import TransferPrepaidCardHandlerABI from './abi/v0.9.0/transfer-prepaid-card-handler';
-import SplitPrepaidCardHandlerABI from './abi/v0.9.0/split-prepaid-card-handler';
-import SpendABI from './abi/v0.9.0/spend';
-import MerchantManagerABI from './abi/v0.9.0/merchant-manager';
-import DeprecatedMerchantManagerABI_0_6_7 from './abi/v0.9.0/deprecated-merchant-manager-0_6_7';
-import RegisterRewardProgramHandlerABI from './abi/v0.9.0/register-reward-program-handler';
-import RegisterRewardeeHandlerABI from './abi/v0.9.0/register-rewardee-handler';
-import SupplierManagerABI from './abi/v0.9.0/supplier-manager';
+import PayMerchantHandlerABI from './abi/v0.10.0/pay-merchant-handler';
+import RegisterMerchantHandlerABI from './abi/v0.10.0/register-merchant-handler';
+import TransferPrepaidCardHandlerABI from './abi/v0.10.0/transfer-prepaid-card-handler';
+import SplitPrepaidCardHandlerABI from './abi/v0.10.0/split-prepaid-card-handler';
+import SpendABI from './abi/v0.10.0/spend';
+import MerchantManagerABI from './abi/v0.10.0/merchant-manager';
+import DeprecatedMerchantManagerABI_0_6_7 from './abi/v0.10.0/deprecated-merchant-manager-0_6_7';
+import RegisterRewardProgramHandlerABI from './abi/v0.10.0/register-reward-program-handler';
+import RegisterRewardeeHandlerABI from './abi/v0.10.0/register-rewardee-handler';
+import SupplierManagerABI from './abi/v0.10.0/supplier-manager';
 
-export const protocolVersions = ['v0.9.0', 'v0.8.7'];
+export const protocolVersions = ['v0.10.0', 'v0.9.0', 'v0.8.7'];
```

Comment:
it should be ok to drop v0.8.7. all the consumers of the SDK are up-to-date now and using teh current version of the protocol which is 0.9.0. there is no need to be backwards compatible to 0.8.7 (so this means dropping all the 0.8.7 ABI's as well, and anything that refers to 0.8.7)